specification.md

```diff
@@ -219,6 +225,14 @@ Optional parameters
 
 Note that the OpenTracing project documents certain **["standard log keys"](./semantic_conventions.md)** which have prescribed semantic meanings.
 
+##### An aside: "Logging" in general, and what it means in OpenTracing
+
+"Logging" is an overloaded term in our industry; one could reasonably argue that all tracing is just a particularly organized form of logging. OpenTracing "logs" are really just key:value maps that describe a particular moment within the context of a Span.
+
+While it's possible to redirect general-purpose process-level logging into OpenTracing, doing so requires care. For instance, logging statements that aren't anchored in specific transactions or traces may not make sense within a tracing system. That said, in enviroments where an overwhelming fraction of conventional logging statements already refer to distributed transactions, `tee`ing that logging data into OpenTracing is reasonable and often beneficial.
```

Comment:
I think adding a warning/note here that says something along the lines of "adding too many logs to a span will increase the size of the span which in turn can have a negative performance impact on application performance and trace pipeline" would be very useful. 

I have seen this issue a few times now in our production environment. Further, this is true for all tracing backends currently, so adding a warning in these docs is appropriate.